src/components/SearchBar.js

```diff
@@ -278,14 +301,26 @@ class SearchBar extends React.Component<SearchBarDefaultProps, SearchBarProps, S
     }
   }
 
+  shareSearch() {
+    const username = AuthUtils.getUserName(AuthUtils.getSavedUser());
+    const signature = `\nfrom \n${username}`;
+    const message = this.props.shareMessage;
+    const searchLink = window.location.href;
+    const emailBody = message + searchLink + signature;
+    const subject = this.props.subject;
+    const emailAddress = this.props.email;
+    document.location.href = `mailto:${emailAddress}?subject=${encodeURIComponent(subject)}&body=${encodeURIComponent(emailBody)}`;
```

Comment:
Could you just use `<a />`, with this string as the `href` instead of having an onClick function?